Added basic new Activities and data

climbz opened the pull request.

climbz committed:
Added basic new Activities and data

alexamster commented on app/src/main/assets/conditioning.xml:
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
I would probs go with json to represent these. much easier to read :)

alexamster commented on app/src/main/java/com/climbz/strongcode/ConditioningActivity.java:
```diff
@@ -0,0 +1,54 @@
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        //my code below
```
remove comment

alexamster commented on app/src/main/java/com/climbz/strongcode/ConditioningActivity.java:
```diff
@@ -0,0 +1,54 @@
+        setSupportActionBar(toolbar);
+
+        //my code below
+        final ListView listview = (ListView) findViewById(R.id.listview);
```
check out http://jakewharton.github.io/butterknife/ , its pretty cool and lets you avoid doing any findviewbyid calls

alexamster commented on app/src/main/java/com/climbz/strongcode/ConditioningActivity.java:
```diff
@@ -0,0 +1,54 @@
+
+        List<Exercise> exercises = null;
+        try {
+            XMLExerciseParserHandler parser = new XMLExerciseParserHandler();
```
using Gson to create objects is much easier then manually parsing xml, hit me up and will explain

alexamster commented on app/src/main/java/com/climbz/strongcode/ConditioningActivity.java:
```diff
@@ -0,0 +1,54 @@
+            public void onItemClick(AdapterView<?> parent, final View view,
+                                    int position, long id) {
+                final String item = (String) parent.getItemAtPosition(position);
+
```
nit: extra whitespace

alexamster commented on app/src/main/java/com/climbz/strongcode/FlexibilityActivity.java:
```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class FlexibilityActivity extends AppCompatActivity {
```
this looks very similar to ConditioningActivity, should probs be combined into one activity

alexamster commented on app/src/main/java/com/climbz/strongcode/GymnasticStaticsActivity.java:
```diff
@@ -0,0 +1,58 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class GymnasticStaticsActivity extends AppCompatActivity {
```
same here.

climbz committed:
Cleanup

Some cleanup in response to comments.
TODO: JSON, GSON, and perhaps butterknife as referenced in Alex's pull
request comments
